README.md

```diff
@@ -1,5 +1,22 @@
 # Screens
 
+This application serves real-time information to all of our browser-based screens posted at stops and stations around the system.
+
+If you're looking for the application that controls the LED countdown clocks and the in-station PA system, please see [mbta/realtime_signs](https://github.com/mbta/realtime_signs).
```

Comment:
Probably would be more accurate to reference [Signs UI](https://github.com/mbta/signs_ui) here (or both)